src/renderer/features/wallets/ForgetWallet/ui/ConfirmForgetModal.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect } from 'react';
+
+import { Wallet } from '@shared/core';
+import { ConfirmModal, FootnoteText, HeaderTitleText } from '@shared/ui';
+import { useI18n } from '@app/providers';
+import { forgetWalletModel } from '../model/forget-wallet-model';
+import { walletUtils } from '@entities/wallet';
+
+type Props = {
+  wallet: Wallet;
+  isOpen: boolean;
+  onClose: () => void;
+  onForget: () => void;
+};
+
+export const ConfirmForgetModal = ({ wallet, isOpen, onClose, onForget }: Props) => {
+  const { t } = useI18n();
+
+  useEffect(() => {
+    forgetWalletModel.events.callbacksChanged({ onDeleteFinished: onForget });
+  }, [onForget]);
+
+  const confirmForgetWallet = () => {
```

Comment:
Please, point to the case for the WalletConnect wallet or Nova Wallet wallet. There should be WalletConnect session removal.

Reply:
It was Implemented before by Alexandr. You can check it in `src/renderer/widgets/WalletDetails/model/wc-details-model.ts` effector model. I checked that after forgetting nova wallet  WC session count in nova wallet goes down